bin/setup

```diff
@@ -1,20 +1,27 @@
 #!/usr/bin/env ruby
```

Comment:
@Robertfarb I see, these files changed during set up.  Can you restore the files that changed during set up to the original files so that they don't show up in the diff?  I usually do something like 'git checkout origin/master bin/setup', which will checkout the version of a single file from origin/master.